tensorflow/contrib/tensorrt/convert/convert_nodes.cc

```diff
@@ -2693,8 +2693,13 @@ tensorflow::Status ConvertGraphDefToEngine(
     VLOG(1) << "Converting op name=" << node_name << ", op=" << node_def.op();
     if (tensorflow::str_util::StartsWith(node_name, kInputPHName) &&
         (node_def.op() == "Placeholder")) {
-      nvinfer1::DimsCHW input_dim_pseudo_chw;
-      for (int i = 0; i < 8; i++) input_dim_pseudo_chw.d[i] = 0;
+      nvinfer1::DataType dtype(nvinfer1::DataType::kFLOAT);
```

Comment:
This conversion is already done in ValidateInputProperties(), I guess you may need to sync with master to merge to changes from PR #20755.